src/app/(proper_react)/redesign/(authenticated)/user/dashboard/fix/data-broker-profiles/view-data-brokers/AboutBrokersIcon.tsx

```diff
@@ -0,0 +1,87 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+"use client";
+
+import { useRef } from "react";
+import { useButton, useOverlayTrigger } from "react-aria";
+import { useOverlayTriggerState } from "react-stately";
+import Image from "next/image";
+import styles from "./AboutBrokersIcon.module.scss";
+import DialogIllustration from "./images/Exploring on Laptop.svg";
+import { QuestionMarkCircle } from "../../../../../../../../components/server/Icons";
+import { useL10n } from "../../../../../../../../hooks/l10n";
+import { ModalOverlay } from "../../../../../../../../components/client/dialog/ModalOverlay";
+import { Dialog } from "../../../../../../../../components/client/dialog/Dialog";
+import { Button } from "../../../../../../../../components/server/Button";
+
+export const AboutBrokersIcon = () => {
+  const l10n = useL10n();
+  const overlayTriggerState = useOverlayTriggerState({
+    defaultOpen: false,
+  });
+  const { triggerProps, overlayProps } = useOverlayTrigger(
+    { type: "dialog" },
+    overlayTriggerState
+  );
+  const triggerRef = useRef<HTMLButtonElement>(null);
+  const { buttonProps } = useButton(triggerProps, triggerRef);
+
+  return (
+    <>
+      <button
+        {...buttonProps}
+        className={styles.triggerButton}
+        ref={triggerRef}
+      >
+        <QuestionMarkCircle
+          alt={l10n.getString(
+            "fix-flow-data-broker-profiles-view-data-broker-profiles-more-dialog-trigger-label"
+          )}
+          width={18}
+          height={18}
```

Comment:
Probably irrelevant, but I get paid by the GitHub comment, so here we are&hellip;
Looks like this is [at least] the third distinct `width`/`height` for the QuestionMarkCircle icon.
(formerly 13&times;13, or 15&times;15; and now 18&times;18)

```sh
git grep "QuestionMarkCircle" | grep -E "(width|height)="

src/app/components/client/ExposureCard.tsx:          icon={<QuestionMarkCircle alt="" width="13" height="13" />}
src/app/components/client/ExposureCard.tsx:            icon={<QuestionMarkCircle alt="" width="13" height="13" />}
src/app/components/client/ExposureCard.tsx:            icon={<QuestionMarkCircle alt="" width="13" height="13" />} // default icon for categories without a unique one
src/app/components/client/ProgressCard.tsx:          <QuestionMarkCircle alt="" width="15" height="15" />
```